optuna/trial.py

```diff
@@ -146,6 +157,72 @@ def _init_relative_params(self):
         self.relative_params = self.study.sampler.sample_relative(self.study, trial,
                                                                   self.relative_search_space)
 
+    def suggest_float(self, name, low, high, log=False):
```

Comment:
Please add a description for `log` in docstring.

Reply:
Added in df6a469.